wizard.html

```diff
@@ -36,9 +36,10 @@ <h1>TCTemplate: TrailO Timed Control Map Cards Generator</h1>
         <li>Add the (single) control to the course. It doesn't matter whether a start and finished are also added, so long as they are outside the map excerpts.</li>
         <li>Move the purple number labelling the control out of the field of view. (At the time of writing, Purple Pen does not permit this label to be removed, which would be preferable here.)</li>
         <li>Set the print area for all courses (under the File menu) to have 0 margin, limit the printable area to the page size and untick <em>Set Print Area Automatically</em>. Select a paper size (custom size if necessary) that is large enough to include all parts of the map that are to be printed. Move the print area to cover all desired parts of the map and click done. (Each course/map does not need to have the same print area, although the author considers it easier to do so.)</li>
-        <li>Create a single PDF of all the courses using File->Create PDFs->Courses and choosing to have one copy of each course all in a single PDF. Select all courses, including those not intended to be printed, but not the all controls map. Select <em>CMYK</em> as the colour model. Upload the maps PDF: <input type="file" id="coursePDFSelector" accept=".pdf" /></li>
-        <li>Create a single PDF of all the control descriptions using File->Create PDFs->Descriptions. Choose fill one page with multiple copies of each course. Select all courses, including those not intended to be printed, but not the all controls. Choose a PDF printer with A4 page size. Leave the box size at the default of 7 mm, and leave the page margins at the default of 12.7 mm on all sides (appears as 13 mm in the main dialog). Upload the descriptions PDF: <input type="file" id="CDPDFSelector" accept=".pdf" /></li>
-        <li>Save the Purple Pen file, then upload it to this wizard: <input type="file" id="ppenSelector" accept=".ppen" onchange="tcTemplate.loadppen(this)" /><input type="button" value="Reload" onclick="tcTemplate.loadppen(document.getElementById('ppenSelector'))" /></li>
+        <li>Create a single PDF of all the courses using File->Create PDFs->Courses and choosing to have one copy of each course all in a single PDF. Select all courses, including those not intended to be printed, but not the all controls map. Select <em>CMYK</em> as the colour model. 
+<p>Upload the <strong>maps</strong> PDF: <input type="file" id="coursePDFSelector" accept=".pdf" /></p></li>
```

Comment:
I don't like the style of a <p> for part of an <li>; use <br /> instead.

Reply:
up to you.  I don't like <br/> but it's your code.

Reply:
Also, the use of `<strong>` looks good here. As part of having a good house style, what do you think is best for describing settings? I have previously used `<em>` but I'm no longer convinced I have used the best tag for the job.

Reply:
I was finding very difficult to see the buttons. Then It was difficult to see which was which. In this font em doesn’t show up as much as strong.